app/models/organization.rb

```diff
@@ -230,7 +230,12 @@ def get_obs_general_calls(options = {})
   def get_twitter_imports_count(options = {})
     date_from, date_to = quota_dates(options)
 
-    SearchTweet.get_twitter_imports_count(users_dataset.join(:search_tweets, :user_id => :id), date_from, date_to)
+    org_search_tweets = Sequel::Model.db.fetch("
+      SELECT * FROM search_tweets st
```

Comment:
Do you really need select * here??
We are not having any tweets usage for org users currently but down the road we will, hence asking..

Reply:
This change is mainly to enforce that the inner join with `search_tweets` is always performed, which should always perform 0 rows.  We should eventually refactor `SearchTweet.get_twitter_imports_count` to manually construct the entire query but I think that change is lower priority given that we currently don't use `search_tweets`.

Reply:
Agreed.. We can take care of such things when we use twitter..